programs/develop/commands/commands-lib/generate-extension-types.ts

```diff
@@ -14,7 +14,7 @@ export async function generateExtensionTypes(projectPath: string) {
   const typePath =
     process.env.EXTENSION_ENV === 'development'
       ? path.resolve(process.cwd(), 'programs/cli/types')
-      : 'extension/dist/types'
+      : './node_modules/extension/dist/types'
```

Comment:
does reverting this change work?

```suggestion
      : 'extension/dist/types'
```

that was part B of [my comment](https://github.com/extension-js/extension.js/pull/165#issuecomment-2330860718):

> B) in the `extension-env.d.ts`, there are paths to `./node_modules/extension/...` - I think you can use `extension/...` with those, without `./node_modules`

Reply:
works indeed! should be live in v2.0.0-alpha.23